Cython/Utility/ModuleSetupCode.c

```diff
@@ -7,6 +7,14 @@
   #define CYTHON_LIMITED_API 1
 #endif
 
+// For the limited API it often makes sense to use Py_LIMITED_API rather than PY_VERSION_HEX
+// when doing version checks.
+#define __PYX_LIMITED_VERSION_HEX PY_VERSION_HEX
+#ifdef Py_LIMITED_API
+  #undef __PYX_LIMITED_VERSION_HEX
+  #define __PYX_LIMITED_VERSION_HEX Py_LIMITED_API
+#endif
```

Comment:
```suggestion
// For the limited API, it often makes sense to use Py_LIMITED_API rather than
// PY_VERSION_HEX when doing version checks.  Make it available in header files
// but guard against redefinitions when included in Cython modules.

#ifndef __PYX_LIMITED_VERSION_HEX
#define __PYX_LIMITED_VERSION_HEX PY_VERSION_HEX
#ifdef Py_LIMITED_API
  #undef __PYX_LIMITED_VERSION_HEX
  #define __PYX_LIMITED_VERSION_HEX Py_LIMITED_API
#endif
#endif
```